src/gatsby-theme-conference/components/speakers.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Link } from 'gatsby'
+import { Styled } from 'theme-ui'
+import { Box, Container } from 'theme-ui/layout'
+import SectionHeading from '../../../node_modules/gatsby-theme-conference/src/components/section-heading'
+import FlexList from '../../../node_modules/gatsby-theme-conference/src/components/flex-list'
+import SpeakerCard from '../../../node_modules/gatsby-theme-conference/src/components/speaker-card'
+
+export default ({speakers = [] }) =>
+  <Box
+    id='speakers'
+    py={5}>
+    <Container>
+      <SectionHeading>
+        Speakers
+      </SectionHeading>
+      <FlexList
+        css={{
+          justifyContent: 'center',
+        }}>
+        {speakers.sort(() => 0.5 - Math.random())
```

Comment:
This should work. What if we keep the keynotes at the top and slice them then randomize the rest. 

I realize I am just adding more and more complexity 😁